internal/services/controller/controller.go

```diff
@@ -108,7 +108,7 @@ func CollectSingleSnapshot(ctx context.Context,
 	f := informers.NewSharedInformerFactory(clientset, 0)
 	df := dynamicinformer.NewDynamicSharedInformerFactory(dynamicClient, 0)
 
-	defaultInformers := getDefaultInformers(f)
+	defaultInformers := getDefaultInformers(f, "")
```

Comment:
question: why do we provide an empty string as a castware namespace here?

Reply:
This is used in the offline snapshot dump, so we don't have a namespace, I'll move it to make it more obvious